src/components/Calculator.js

```diff
@@ -1,31 +1,44 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 /* eslint-disable */
 
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {total: 0, next: null, operation: null}
+  }
+
+  handleClick = (event) => {
+    const buttonName = event.target.innerText;
+    const result = calculate(this.state, buttonName);
+    this.setState(result)
+  }
+  
   render() {
+    const {total, next, operation } = this.state;
     return (
       <div className="container">
         <div className="wrapper">
-          <div className="display">0</div>
-          <div className="btn grey">AC</div>
-          <div className="btn grey">+/-</div>
-          <div className="btn grey">%</div>
-          <div className="btn orange">/</div>
-          <div className="btn grey">7</div>
-          <div className="btn grey">8</div>
-          <div className="btn grey">9</div>
-          <div className="btn orange">x</div>
-          <div className="btn grey">4</div>
-          <div className="btn grey">5</div>
-          <div className="btn grey">6</div>
-          <div className="btn orange">-</div>
-          <div className="btn grey">1</div>
-          <div className="btn grey">2</div>
-          <div className="btn grey">3</div>
-          <div className="btn orange">+</div>
-          <div className="btn grey span">0</div>
-          <div className="btn grey">.</div>
-          <div className="btn orange">=</div>
+          <button className="display">{total}{operation}{next}</button>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindle make sure you update the styling of the **Display Area** to make numbers be visible by users. Below is an image on looks your calculator, inputs numbers are not totally visible.
You can fix it by reducing the value of the `padding-top` in the `.display { .... }` class, you can try `1rem` instead of `3rem`.
![Screenshot from 2022-11-10 14-29-05](https://user-images.githubusercontent.com/83042742/201093106-e08a416c-80fd-43c2-9712-ea7894736ed5.png)